examples/dynamic-bridge-app/linux/main.cpp

```diff
@@ -176,6 +205,127 @@ std::unique_ptr<GeneratedCluster> CreateCluster(chip::ClusterId id)
     return nullptr;
 }
 
+CHIP_ERROR ReadValueFromBuffer(chip::TLV::TLVWriter & wr, const Span<const char> & v)
+{
+    return wr.PutString(chip::TLV::Tag(), v);
+}
+CHIP_ERROR ReadValueFromBuffer(chip::TLV::TLVWriter & wr, const bool & v)
+{
+    return wr.PutBoolean(chip::TLV::Tag(), v);
+}
+
+template <typename T>
+CHIP_ERROR ReadValueFromBuffer(chip::TLV::TLVWriter & wr, const T & v)
+{
+    return wr.Put(chip::TLV::Tag(), v);
+}
+
+CHIP_ERROR WriteValueToBuffer(const bool & value, uint8_t * buffer, const uint16_t & maxReadLength)
```

Comment:
why is masxReadLengh a const reference? Why not just a uint16?

Reply:
Changed `buffer, length` pairs to use `Span` in [b91b1f8](https://github.com/project-chip/connectedhomeip/pull/23840/commits/b91b1f831352c37876d759ee883df7802abe9089).

`maxReadLength` here was interpreted as the max number of bytes to read from `value`. The above commit shows a clearer usage.